README.rst

```diff
@@ -1,4 +1,15 @@
 
 **motionEye** is a web-based frontend for `motion <https://motion-project.github.io>`_. Check out the `wiki <https://github.com/ccrisan/motioneye/wiki>`_ for more details. Changelog is available on the `releases page <https://github.com/ccrisan/motioneye/releases>`_.
 
-Due to personal reasons I can no longer be actively involved with this project. If anyone is interested in taking it over, please contact me and we'll work out together a hand-off plan.
+Due to personal reasons `Calin Crisan <https://github.com/ccrisan>`_ can no longer be actively involved with this project. It has been transferred to the new `motioneye-project <https://github.com/motioneye-project>`_ GitHub organization with a new `developer team <https://github.com/orgs/motioneye-project/people>`_. Please check out the `dev <https://github.com/motioneye-project/motioneye/tree/dev>`_ branch for current development, which includes:
+
+- Migration to Python 3
+- Internationalization
+- Migration to setuptools
+- Many more bug fixes and enhancements

```

Comment:
```suggestion
- [x] Migration to Python 3
- [x] Internationalization
- [x] Migration to setuptools
- [x] Many more bug fixes and enhancements

```

Reply:
Ah dammit, this is not Markdown but reStructuredText, hence the check list cannot be added like that.

Reply:
Reverted. I guess you wanted a real Markdown checklist. in reStructuredText there is no nativ syntax for it, hence I think a regular list is best.